src/Microsoft.Azure.WebJobs.Host/Bindings/SystemBindingData.cs

```diff
@@ -36,14 +36,14 @@ internal class SystemBindingData
         public string MethodName { get; set; }
 
         /// <summary>
-        /// Get the current UTC date. 
+        /// Get the current UTC date. This returns the current value each time it is called.
         /// </summary>
-        public DateTime UtcNow => DateTime.UtcNow;
+        public DateTime DateTime => DateTime.UtcNow;
```

Comment:
>DateTime [](start = 24, length = 8)

We really should include the timezone, just like System.DateTime.UtcNow does. Avoids the confusion and accidentally mistaking this for local time. .